backend/routes/webhooks.ts

```
@@ -0,0 +1,122 @@
+                    break;
+
+                case 'customer.subscription.deleted':
+                    // Update status to 'canceled' for the specific subscription
+                    console.log('Updating subscription status to canceled for:', subscription.id);
+                    const { error: deleteError } = await supabase
+                        .from('subscriptions')
+                        .update({
+                            status: 'canceled' as SubscriptionStatus, // Cast status
+                            updated_at: new Date().toISOString()
+                         })
+                        .eq('stripe_subscription_id', subscription.id);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)